mails/themes/dark_modern/components/header.html.twig

```diff
@@ -0,0 +1,73 @@
+<!--[if mso | IE]>
```

Comment:
Are these comments still useful?

Reply:
This is generated by MJML so I wouldn't touch them
Email integration is really weird ^^